src/main/java/com/gmail/inverseconduit/commands/sets/CoreBotCommands.java

```diff
@@ -179,7 +179,11 @@ private void createNumberCommand() {
     private void createShutdownCommand(ChatInterface chatInterface) {
         CommandHandle shutdown =
                 new CommandHandle.Builder("shutdown", message -> {
-                    chatInterface.broadcast("*~going down*");
+                    final String bcMessage = message.getMessage().replaceFirst(Pattern.quote(BOT_CONFIG.getTrigger()) + "shutdown", "");
```

Comment:
That should be already given by the way commands are invoked, but sure thing, we definitely can